### PR TITLE
Update document:  add how to npm i maplibre-gl on intro page

### PR DESCRIPTION
**Pull request by naogify**
## Summary
Related to https://github.com/maplibre/maplibre-gl-js/issues/4228. I added how to npm install maplibre-gl on intro page. 

## Launch Checklist

 - [x] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license) - if you are not sure about this, please ask!
 - [x] Briefly describe the changes in this PR.
 - [x] Link to related issues.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality.
 - [ ] Document any changes to public APIs.
 - [ ] Post benchmark scores.
 - [x] Add an entry to `CHANGELOG.md` under the `## main` section.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -74,3 +74,30 @@ You can view a listing of all the files in the MapLibre GL JS package by appendi
 | :------- | :---: | :----: |
 | `latest` | <https://unpkg.com/maplibre-gl/dist/maplibre-gl.js> | <https://unpkg.com/maplibre-gl/dist/maplibre-gl.css> |
 | Use at least `2.4.x` | <https://unpkg.com/maplibre-gl@^2.4/dist/maplibre-gl.js> | <https://unpkg.com/maplibre-gl@^2.4/dist/maplibre-gl.css> |
+
+
+## NPM
```
Comment (HarelM) [inline docs/index.md:79]: I would consider moving this to upwards, it's the most common usage, before CDN, CSP and other less common techniques...

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,7 @@
 - "Accept" headers set in Request Transformers are not overwritten [#4210](https://github.com/maplibre/maplibre-gl-js/pull/4210)
 - ⚠️ Rename projMatrix to modelViewProjectionMatrix. Also rename invProjMatrix, alignedProjMatrix accordingly [#4215](https://github.com/maplibre/maplibre-gl-js/pull/4215)
 - Publish an unminified prod build [#4265](https://github.com/maplibre/maplibre-gl-js/pull/4265)
+- Add a section to the documentation about using MapLibre GL JS with npm ([#4271](https://github.com/maplibre/maplibre-gl-js/pull/4271))
```
Comment (HarelM) [inline CHANGELOG.md:15]: I don't think this is interesting... I would recommend removing this from the changelog.

File: docs/index.md (hunk 1 above)
Comment (naogify) [reply docs/index.md:79]: Thank you for review. I agree with npm is most common usage. How about moving npm section after "quick start" section, or after "Reading this documentation" section.  

After Quick Start Section.
スクリーンショット 2024-06-14 20 27 28

After Reading this documentation section.
スクリーンショット 2024-06-14 20 31 24

File: CHANGELOG.md (hunk 2 above)
Comment (naogify) [reply CHANGELOG.md:15]: Thank you for the review. I was unsure whether to include it, so I will remove it.

File: docs/index.md (hunk 1 above)
Comment (HarelM) [reply docs/index.md]: After reading... The second one is better I believe.

Comment (naogify) [reply docs/index.md]: I understand. I have moved the npm section to the end of  "After Reading.." section. 
Could you check it?

**Review by HarelM — APPROVED** [review]

**Pull request merged**